_commands/basics/kill.md

```diff
@@ -0,0 +1,58 @@
+---
+---
+
+kill
+--
+
+`kill` is used to send signal to a process or to kill a process.
+We typically use kill PID, where you know the PID of the process.
+By default `kill` generates SIGTERM, to safely end a process.
+
+
+~~~ bash
+$ kill 7243
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `kill [-s] %pid`
+~~~ bash
+$ ps -ef | grep vim
+username    7243  7222  9 22:43 pts/2    00:00:00 vim
+
+$ kill -9 7243
+~~~
+
+##### Break it down
+ * `ps -ef` will print out all the processes running on the user's machine.
+ Piping with `| grep vim` will only output processes with the name
+ given, in this case only vim processes are printed. `kill -9 7243` is using
+ SIGTERM -9 to send the end process (kill) signal to process 7243, which
```

Comment:
`kill -9` sends `SIGKILL` not `SIGTERM -9`. Emphasize you should only use `-9` if a process is non-responsive to a normal `kill`.